src/mds/StrayManager.cc

```diff
@@ -369,18 +369,6 @@ void StrayManager::advance_delayed()
       dout(4) << __func__ << ": delayed dentry is now null: " << *dn << dendl;
       continue;
     }
-
-    const bool purging = eval_stray(dn);
```

Comment:
Can we actually remove this line? I think this causes these errors and others: /ceph/teuthology-archive/pdonnell-2019-03-30_05:01:26-fs-wip-pdonnell-testing-20190329.214758-distro-basic-smithi/3788265/teuthology.log